communityCustomizations/ShowPasswordButton/OnLoad.js

```diff
@@ -0,0 +1,15 @@
+var inputArea = document.getElementById("inputArea");
+
+if(inputArea){
+     console.log("hello");
+     var showButton = document.createElement("div");
+ showButton.id = "showButton";  
+ showButton.innerHTML = "Show password";
+ inputArea.appendChild(showButton);
```

Comment:
In your screenshots, it looks like you have the nice image in the textbox to show the password. Is that done with CSS?

Reply:
I have Centered UI applied for my adfs. Still not sure, if the button originates from there